frontend/tsconfig.json

```
@@ -15,6 +15,11 @@
     "incremental": true,
     "sourceMap": true,
     "inlineSources": true,
+    
+    "strict": true, 
```

Biome (2.1.2)

[error] The key strict was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)